Sources/SwiftCoAP/SCClient.swift

```diff
@@ -11,23 +11,23 @@ import UIKit
 //MARK:
 //MARK: SC Client Delegate Protocol declaration
 
-@objc protocol SCClientDelegate {
+public protocol SCClientDelegate: AnyObject {
```

Comment:
We need `AnyObject` for it to be able to be `weak`, right?

Reply:
Yup.